docs/source/api/v1/deliveryservice_requests.rst

```diff
@@ -0,0 +1,972 @@
+            "cdnName": null,
+            "checkPath": null,
+            "displayName": "Demo 1",
+            "dnsBypassCname": null,
+            "dnsBypassIp": null,
+            "dnsBypassIp6": null,
+            "dnsBypassTtl": null,
+            "dscp": 0,
+            "edgeHeaderRewrite": null,
+            "geoLimit": 0,
+            "geoLimitCountries": null,
+            "geoLimitRedirectURL": null,
+            "geoProvider": 0,
+            "globalMaxMbps": null,
+            "globalMaxTps": null,
+            "httpBypassFqdn": null,
+            "id": 1,
+            "infoUrl": null,
+            "initialDispersion": 1,
+            "ipv6RoutingEnabled": true,
+            "lastUpdated": "2019-05-15 14:32:05+00",
+            "logsEnabled": true,
+            "longDesc": "Demo Delivery Service Request",
+            "longDesc1": null,
+            "longDesc2": null,
+            "matchList": [
+                {
+                    "type": "HOST_REGEXP",
+                    "setNumber": 0,
+                    "pattern": ".*\\.demo1\\..*"
+                }
+            ],
+            "maxDnsAnswers": null,
+            "midHeaderRewrite": null,
+            "missLat": 42,
+            "missLong": -88,
+            "multiSiteOrigin": false,
+            "originShield": null,
+            "orgServerFqdn": "http://origin.infra.ciab.test",
+            "profileDescription": null,
+            "profileId": null,
+            "profileName": null,
+            "protocol": 2,
+            "qstringIgnore": 0,
+            "rangeRequestHandling": 0,
+            "regexRemap": null,
+            "regionalGeoBlocking": false,
+            "remapText": null,
+            "routingName": "video",
+            "signed": false,
+            "sslKeyVersion": null,
+            "tenantId": 1,
+            "type": "HTTP",
+            "typeId": 1,
+            "xmlId": "demo1",
+            "exampleURLs": [
+                "http://video.demo1.mycdn.ciab.test",
+                "https://video.demo1.mycdn.ciab.test"
+            ],
+            "deepCachingType": "NEVER",
+            "fqPacingRate": null,
+            "signingAlgorithm": null,
+            "tenant": "root",
+            "trResponseHeaders": null,
+            "trRequestHeaders": null,
+            "consistentHashRegex": null,
+            "consistentHashQueryParams": [
+                "abc",
+                "pdq",
+                "xxx",
+                "zyx"
+            ],
+            "maxOriginConnections": 0,
+            "ecsEnabled": false
+        },
+        "status": "complete"
+	}]}
+
+
+``POST``
+========
+Allows users to create :ref:`ds_requests`.
+
+:Auth. Required: Yes
+:Roles Required: Portal\ [#tenancy]_
+:Response Type:  Array
+
+Request Structure
+-----------------
+:changeType:           The type of change made to the delivery service as described in this :ref:`ds_requests`
+:status:               The status of this :ref:`ds_requests`
```

Comment:
Is this the Name of a Status? Or an ID of a Status? Or something else altogether?